lms/services/d2l_api/client.py

```diff
@@ -121,11 +122,11 @@ def group_set_groups(self, org_unit, group_category_id, user_id=None):
 
         return groups
 
-    def list_files(self, org_unit) -> list[dict]:
+    def list_files(self, course_id) -> list[LMSDocument]:
```

Comment:
Rename the parameter to avoid pylint complaining.  Org unit is 100% equivalent to a course in this call anyway.

Reply:
Where did the term "org unit" come from? Does it ever refer to something other than a course in our code?